helm-charts/yunikorn/templates/deployment.yaml

```diff
@@ -104,3 +111,10 @@ spec:
               protocol: TCP
           resources:
             {{- toYaml .Values.web.resources | nindent 12 }}
+      {{- if .Values.enableGoCoverDir }}
+      volumes:
+        - name: go-cover-dir
+          hostPath:
+            path: /go-cover-dir/
+            type: DirectoryOrCreate
+      {{- end }}
```

Comment:
If follow this approach, I believe we'll need extra code somewhere which fetches the results from the Docker containers if we want to automate this, correct? Because we won't have it right away on the host filesystem. I guess we need a `docker cp` somewhere.

Reply:
Yes, like `copy_go_cover_dir` function in https://github.com/apache/yunikorn-k8shim/pull/733/files#diff-6d485e1346af49aeb05caeb7af55511940214789cd6976f03afa05e56b82d639R156-R177.